src/components/BorrowerProfile/RepaymentSchedule.vue

```diff
@@ -0,0 +1,233 @@
+<template>
+	<div>
+		<button class="tw-text-h4 tw-text-link tw-mt-3"
+			@click="openLightbox"
+			v-if="this.status === 'fundraising'"
+		>
+			Detailed repayment schedule >
+		</button>
+		<kv-lightbox
+			:visible="isLightboxVisible"
+			title="Loan repayment schedule"
+			@lightbox-closed="closeLightbox"
+		>
+			<!-- Field Partner loan fundraising -->
+			<div v-if="isPartnerLoan">
+				<p class="tw-inline-block tw-pb-3">
+					Repayments {{ statusLanguageCheck }} in
+				</p>
+				<p class="tw-inline-block tw-font-medium ">
+					{{ formattedFirstRepaymentDate }}
+				</p>
+				<span v-if="this.status === 'payingBack'">
+					<p class="tw-inline-block">
+						and are
+					</p>
+					<p class="tw-inline-block tw-font-medium">
+						{{ repaymentStatusCheck }}.
+					</p>
+				</span>
+				<table class="tw-table-auto">
+					<tr class="tw-bg-secondary tw-text-left">
+						<th></th>
```

Comment:
Is there a reason we don't label this table header? Maybe `<th>Date</th>` or something like that?

Reply:
It wasn't labeled in the spec, so I left it out. 
<img width="452" alt="Screen Shot 2021-12-08 at 2 53 37 PM" src="https://user-images.githubusercontent.com/1521381/145290457-fa835125-9d24-401e-9f95-26021f77502d.png">

Reply:
OK. Well, we have some text in there for screen readers at a minimum. 
`<th><span class="tw-sr-only">Date</span></th>` should do it.